layouts/_default/baseof.html

```diff
@@ -9,7 +9,7 @@
 
   {{- partial "global_navbar.html" . -}}
   {{ $path := split $.Source.File.Path .Site.Params.PathSeparator }}
-  {{ if or (eq (index $path 0) "events") (eq .Type "events") }}
```

Comment:
I totally don't understand why this is working.

It's checking to see if the path is "events", which should match against both `/events/2017-chicago` and `/events` pages. Ooh. Never mind. The path is filesystem path, so that's why this works.